packages/core-bridge/index.d.ts

```diff
@@ -3,6 +3,20 @@ import type { TLSConfig } from '@temporalio/common/lib/internal-non-workflow';
 
 export { TLSConfig };
 
+type Shadow<Base, New> = {
```

Comment:
That's quite the type..

Reply:
I originaly intended to name it `RecursiveReplace`, but we already have a `NestedReplace` elsewhere, which is something completely different; that would have been confusing. But I believe it could makes things much easier than `Replace<Base, New>` in many cases.